unit_tests/testParameterLists/testParameterLists.f90

```diff
@@ -4634,6 +4635,113 @@ SUBROUTINE testGetString()
+        '3.200000000000000E+01; 4.200000000000000E+01')
+    ASSERT(tmpbool,'testSDKa2_2 string')
+    CALL testParam%getString('testPL->testSNKa2',tmpstr)
+    tmpbool=(tmpstr == '11; 21; 12; 22')
+    ASSERT(tmpbool,'testSNKa2 string')
+    CALL testParam%getString('testPL->testSNKa2_2',tmpstr)
+    tmpbool=(tmpstr == '31; 41; 32; 42')
+    ASSERT(tmpbool,'testSNKa2_2 string')
+    CALL testParam%getString('testPL->testSLKa2',tmpstr)
+    tmpbool=(tmpstr == '110; 210; 120; 220')
+    ASSERT(tmpbool,'testSLKa2 string')
+    CALL testParam%getString('testPL->testSLKa2_2',tmpstr)
+    tmpbool=(tmpstr == '310; 410; 320; 420')
+    ASSERT(tmpbool,'testSLKa2_2 string')
+    CALL testParam%getString('testPL->testSTRa2',tmpstr)
+    tmpbool=(tmpstr == 'stringarray1; stringarray2; stringarray3; stringarray4')
+    ASSERT(tmpbool,'testSTRa2 string')
+    CALL testParam%getString('testPL->testSTRa2_2',tmpstr)
+    tmpbool=(tmpstr == 'stringarray5; stringarray6; stringarray7; stringarray8')
+    ASSERT(tmpbool,'testSTRa2_2 string')
+
+    !Test 3-D arrays
+    CALL testParam%getString('testPL->testSSKa3',tmpstr)
+    tmpbool=(tmpstr == '1.110000E+00; 2.110000E+00; 1.210000E+00; 2.210000E+00; '// &
+        '1.120000E+00; 2.120000E+00; 1.220000E+00; 2.220000E+00')
+    ASSERT(tmpbool,'testSSKa3 string')
+    CALL testParam%getString('testPL->testSSKa3_2',tmpstr)
+    tmpbool=(tmpstr == '3.110000E+00; 4.110000E+00; 3.210000E+00; 4.210000E+00; '// &
+        '3.120000E+00; 4.120000E+00; 3.220000E+00; 4.220000E+00')
+    ASSERT(tmpbool,'testSSKa3_2 string')
+    CALL testParam%getString('testPL->testSDKa3',tmpstr)
+    tmpbool=(tmpstr == '1.110000000000000E+01; 2.110000000000000E+01; '// &
+        '1.210000000000000E+01; 2.210000000000000E+01; '// &
+        '1.120000000000000E+01; 2.120000000000000E+01; '// &
+        '1.220000000000000E+01; 2.220000000000000E+01')
+    ASSERT(tmpbool,'testSDKa3 string')
+    FINFO() CHAR(tmpstr)
+    CALL testParam%getString('testPL->testSDKa3_2',tmpstr)
+    tmpbool=(tmpstr == '3.110000000000000E+01; 4.110000000000000E+01; '// &
+        '3.210000000000000E+01; 4.210000000000000E+01; '// &
+        '3.120000000000000E+01; 4.120000000000000E+01; '// &
+        '3.220000000000000E+01; 4.220000000000000E+01')
+    ASSERT(tmpbool,'testSDKa3_2 string')
+    CALL testParam%getString('testPL->testSNKa3',tmpstr)
+    tmpbool=(tmpstr == '111; 211; 121; 221; 112; 212; 122; 222')
+    ASSERT(tmpbool,'testSNKa3 string')
+    CALL testParam%getString('testPL->testSNKa3_2',tmpstr)
+    tmpbool=(tmpstr == '311; 411; 321; 421; 312; 412; 322; 422')
+    ASSERT(tmpbool,'testSNKa3_2 string')
+    CALL testParam%getString('testPL->testSLKa3',tmpstr)
+    tmpbool=(tmpstr == '111; 211; 121; 221; 112; 212; 122; 222')
+    ASSERT(tmpbool,'testSLKa3 string')
+    CALL testParam%getString('testPL->testSLKa3_2',tmpstr)
+    tmpbool=(tmpstr == '311; 411; 321; 421; 312; 412; 322; 422')
+    ASSERT(tmpbool,'testSLKa3_2 string')
```

Comment:
Should really be using the new unit test macros... so the above 2 lines become `ASSERT_EQ(tmpstr, '311; 411; 321; 421; 312; 412; 322; 422','testSLKa3_2 string')`.  Switch to those throughout the test instead of using `tmpbool`.  The main times it's ok not to use the new macros is when an `ANY(...)` or `ALL(...)` statement is required for testing.

Reply:
Or when there is a comparison that extends beyond the 80 char limit I assume as well, right?

Reply:
The 80 character limit is defunct now.  We typically cap at 100 now (and allow beyond 100 if there's a good reason for it, though that's not too common).  But yeah, long lines with the macros are also exceptions.